src/katgpucbf/fgpu/ddc.py

```diff
@@ -44,42 +44,44 @@ class DDCTemplate:
         The GPU context that we'll operate in
     taps
         Number of taps in the FIR filter
-    decimation
+    subsampling
         Fraction of samples to retain after filtering
```

Comment:
I know you've updated fgpu.design, but worth asking if the docstring needs an update? As per your description on the ticket, subsampling is
> the factor by which the time between samples is increased by discarding samples [...]

Is this still implied by the existing docstring/parameter explainer?

Reply:
It's the same thing. If there was a sample every 1ns before, and we retain 1/8 of the samples, then (assuming that means we retain every 8th sample) there will be a sample every 8ns afterwards.